sopel_8ball/choices.py

```diff
@@ -118,3 +118,37 @@ def choices(self):
             "Would you take ‘no’ for an answer?",
             "Not in this lifetime!",
         )
+
+
+class Spooky(AbstractChoiceProvider):
+    """A spooky magic 8 ball."""
+    def choices(self) -> Tuple[str, ...]:
+        return (
+            # affirmative answers
+            "The ghost over your shoulder said yes.",
+            "The accursed screams in agreement.",
+            "The elders smile from the deep.",
+            "I've seen this in the dreams of a Great Old One.",
+            "For once eerie voices whisper in harmony.",
+
+            # non-committal answers
+            "The abyss remain silent.",
+            "I hear the whispers of a thousand voices "
+            "but none have your answer.",
+            "The unfathomable truth cannot be revealed.",
+            "In the darkness this won't matter anymore.",
+            "From the depth comes no answer.",
+
+            "The spirits are bored of this tune.",
+            "The answer lies behond the stars, ineffable.",
+            "Answers are lost in the midst of black seas of infinity.",
+            "You shall go mad from the revelation of the truth.",
+            "I hear your call yet their shriek keeps me silent.",
```

Comment:
```suggestion
            "I hear your call, yet their shrieking keeps me silent.",
```

This is stylistic, not necessarily grammatical.